src/pages/error.vue

```diff
@@ -31,19 +32,30 @@ export default {
       password: ''
     }
   },
-
+  computed: {
+    ...mapGetters(['configuration'])
+  },
   methods: {
     authenticate () {
-      this.$store.dispatch('authenticate', { provider: 'oauth2' }).then(() => {
-        this.$router.push('/')
-      })
+      this.$store.dispatch('authenticate', { provider: 'oauth2' })
+        .then(() => {
+          this.$router.push({ path: '/' })
+        })
+        .catch(() => {
+          this.$router.push('/error')
+        })
     }
   }
 }
 </script>
 
 <style scoped="true">
-    .background {
-        height: 100%;
-    }
+  .background{
+    height: "100%";
+  }
+  .center-dialog {
```

Comment:
shouldn't this be a generic CSS class for centering ?

Reply:
actually it should be centered by justify-content-center but this didn't work as expected. In this case i could trace back the problem to uikit - so this can be resolved as soon as we can get rid of uikit.

Reply:
mind adding a "FIXME" comment in the code so that the next passersby will not wonder ?

Reply:
added FIXME